tedana/workflows/tedana.py

```diff
@@ -331,6 +344,14 @@ def tedana_workflow(data, tes, out_dir='.', mask=None,
     if not op.isdir(out_dir):
         os.mkdir(out_dir)
 
+    io.outdir = out_dir
+    if prefix and prefix[-1] != '_':
+        prefix += '_'
+    io.prefix = prefix
+    if convention == 'bids':
+        convention = constants.bids
+    io.convention = convention
```

Comment:
```suggestion
    io.convention = io.set_convention(convention)
```

I created a new function, io.set_convention so that more logic can be used to parse user inputted convention options into a smaller list of options that are used in constants.py. Right now, it includes the current two options, plus a way to input a tedana version number, which would then convert it to the correct convention name. Note that the choices in argparse won't allow for a version number to be entered & I'm not sure how to allow both fixed choices and a more flexible choice within argparse. This is current irrelevant, since there are only two options for conventions, but I figured it's better to modularization this now to allow for easier expansion later on.